pages/service/[id].tsx

```diff
@@ -459,7 +437,7 @@ export const ServicePage = ({ serviceData }: ServicePageProps): JSX.Element => {
       </Section>
 
       <Footer>
-        <ButtonLink as="button" onClick={() => router.back()} type="button">
+        <ButtonLink as="button" onClick={() => router.push('/')} type="button">
```

Comment:
This is intentional. The back buttons here has to return us to either the `/`, `/saved`, `/quiz/results` without knowing which one we've come from.

Reply:
Oh ok, maybe we need to store previous page somehow to guard against the scenario @jthistle mentioned above. I'll change it back for now.

Reply:
It's not ideal, but better than going to the wrong place. It was mentioned when this went in, I'll make sure it's actually carded.